nevergrad/optimization/optimizerlib.py

```diff
@@ -1788,12 +1809,12 @@ class EMNA(base.ConfiguredOptimizer):
 
     # pylint: disable=unused-argument
     def __init__(
-        self,
-        *,
-        isotropic: bool = True,
-        naive: bool = True,
-        population_size_adaptation: bool = False,
-        initial_popsize: tp.Optional[int] = None,
+            self,
```

Comment:
not sure this is a good idea; why adding spacing here ?

Reply:
a linter did it automatically, it's alright (there are 2 conventions for spacing in this situations...)  although it makes reading the PR a bit more difficult